src/main/java/org/lightcouch/View.java

```diff
@@ -337,12 +337,12 @@ public boolean queryForBoolean() {
         }
     }
 
-    private <T> Page<T> queryPage(PagingDirection direction, int rowsPerPage, String
+    private <T> Page<T> queryPage(PagingDirection direction, int rowsPerPage, JsonElement
             currentStartKey,
-                                  String currentStartKeyDocId, String startKey, String
+                                  String currentStartKeyDocId, JsonElement startKey, String
                                           startKeyDocId, Class<T> classOfT) {
         // set view query params
-        limit(rowsPerPage + 1);
+        limit(rowsPerPage + 1, true);
```

Comment:
I think this should just be `limit(rowsPerPage +1);` unless someone can convince me that there is a views case where we need to specify multiple limit parameters.